discovery/gossiper.go

```diff
@@ -2344,13 +2344,63 @@ func (d *AuthenticatedGossiper) processNetworkAnnouncement(
 				source: nMsg.source,
 				msg:    e1Ann,
 			})
+
+			// In addition to sending this edge's policy, we'll also
+			// attempt to send the backing node's announcement if we
+			// happen to know about it.
+			pubKey, _ := chanInfo.NodeKey1()
```

Comment:
How is this change related to the rest of the PR? Could it be made its own PR?

Reply:
It's related in that it solves some integration test issues that became more apparent with the cache enabled. Not opposed to making it its own PR - let me know what you think.

Reply:
Yeah, let's make it a own PR, should be fairly easy to review and get in.

Reply:
It should also have an accompanying test to demonstrate the problem.

Reply:
Turns out these changes are actually no longer needed due to #1981!